flocker/node/agents/testtools.py

```diff
@@ -4,60 +4,14 @@
 Test helpers for ``flocker.node.agents``.
```

Comment:
I'm not clear on the distinction between this and `.test.blockdevicefactory`. I wonder if this should be a package with both these as submodules?